card/spec/lib/card/set_pattern_spec.rb

```diff
@@ -18,21 +18,21 @@ def it_generates( opts )
 #FIXME - these should probably be in pattern-specific specs, though that may not leave much to test in the base class :)
 
 describe Card::RightSet do
-  it_generates :name => "author+*right", :from => Card.new( :name => "Iliad+author" )
-  it_generates :name => "author+*right", :from => Card.new( :name => "+author" )
+  it_generates name: "author+*right", from: Card.new( name: "Iliad+author" )
+  it_generates name: "author+*right", from: Card.new( name: "+author" )
```

Comment:
Space inside parentheses detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.